### DISCUSSION TITLE
Why eachWeekendOfInterval don't have weekStartsOn option

### DISCUSSION THREAD

**Discussion by irkreja** [js-fns/date-fns#1831] [General] [answered]
Why not eachWeekendOfInterval didn't support options like locale, weekStartsOn? 

You already have options for eachWeekOfInterval
```eachWeekOfInterval(interval, [options])```

Please add support for options for eachWeekendOfInterval too.
```eachWeekendOfInterval(interval)```

**Comment by kossnocorp** [comment, marked as answer]
Because weekend and week are separate things. Sunday is a weekend regardless of when the week starts, e.g in the US. Given that this option won't have any effect.
